### PR TITLE
Rename the project

### PR DESCRIPTION
**Pull request by revit13**
This PR prepares the code for renaming the project to "fybrik" as noted in issue #712.
This implies that the organization name is changed as well as the repo name and the owner name to "fybrik"
The renamings that were done can be found under the commits tab which contains commit for each string replacment. (for example renaming  m4d to fybrik)
Some of the non trivial renaming are:
github.com/mesh-for-data/mesh-for-data -- > fybrik.io/fybrik in go code, for vanity go import
https://mesh-for-data.github.io/mesh-for-data --> fybrik.io
m4d-ibm-com --> fybrik-io
m4d.ibm.com --> fybrik.io

Part of #712

### CODE REVIEW

File: charts/fybrik/files/webhook-configs.yaml [hunk 1]
```diff
@@ -35,12 +35,12 @@ webhooks:
       service:
         name: webhook-service
         namespace: '{{ .Release.Namespace }}'
-        path: /mutate-motion-m4d-ibm-com-v1alpha1-streamtransfer
```
Comment (ronenkat) [inline charts/fybrik/files/webhook-configs.yaml:38]: m4d-ibm-com -->> fybrik.io

File: charts/fybrik/values.yaml [hunk 2]
```diff
@@ -15,12 +15,12 @@ clusterScoped: true
 
 # Global configuration applies to multiple components installed by this chart
 global:
-  # Default hub for Mesh for Data images.
+  # Default hub for Fybrik images.
   hub: ghcr.io/mesh-for-data
```
Comment (ronenkat) [inline charts/fybrik/values.yaml:19]: should change mesh-for-data as well to Fybrik

File: cmd/root.go [hunk 3]
```diff
@@ -17,7 +17,7 @@ var cfgFile string
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "mesh-for-data",
```
Comment (ronenkat) [inline cmd/root.go:19]: mesh-for-data --> fybrik

File: connectors/egeria/src/main/java/com/ibm/egeria/EgeriaClient.java [hunk 4]
```diff
@@ -13,16 +13,16 @@
...
+import com.fybrik.DataCatalogResponse.CatalogDatasetInfo;
+import com.fybrik.DatasetDetailsOuterClass.CredentialsInfo;
+import com.fybrik.DatasetDetailsOuterClass.DatasetDetails;
+import com.fybrik.DatasetDetailsOuterClass.DatasetMetadata;
+import com.fybrik.DatasetDetailsOuterClass.Db2DataStore;
+import com.fybrik.DatasetDetailsOuterClass.KafkaDataStore;
+import com.fybrik.DatasetDetailsOuterClass.S3DataStore;
+import com.fybrik.DatasetDetailsOuterClass.DataStore.DataStoreType;
+import com.fybrik.DatasetDetailsOuterClass.DataComponentMetadata;
+import com.fybrik.DatasetDetailsOuterClass.DataStore;
```
Comment (ronenkat) [inline connectors/egeria/src/main/java/com/ibm/egeria/EgeriaClient.java:25]: should be io.fybrik

Comment (revit13) [reply connectors/egeria/src/main/java/com/ibm/egeria/EgeriaClient.java:25]: Thanks will fix that.

File: site/mkdocs.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ strict: false
 site_name: Fybrik
 site_description: A cloud-native platform to control data usage
 copyright: Made with ❤️ by Fybrik contributors.
-repo_name: fybrik/fybrik
+repo_name: fybrik.io/fybrik
```
Comment (roee88) [inline site/mkdocs.yml:6]: Is the organization going to be called `fybrik.io`? I thought it would be just fybrik

Comment (revit13) [reply site/mkdocs.yml:6]: This is the name that appears at the top right of the site which links to the github repo.

**Review by ronenkat — APPROVED** [review]

**Pull request merged**